static/explorer.scss

```diff
@@ -880,8 +880,18 @@ html[data-theme=dark] {
     cursor: pointer;
 }
 
-.currentCursorPosition, .ctrlSNothing {
+.ctrlSNothing {
     font-size: small;
     border: none;
     user-select: text;
 }
+
+.currentCursorPosition {
+    position: absolute;
+    bottom: 0;
+    right: 14px; // width of monaco scrollbar
+    font-size: small;
+    color: #17a2b8;
```

Comment:
Tried this locally and I think it looks a bit nicer
```suggestion
    border-radius: 0.5rem;
    background-color: rgba(49, 54, 60, 0.85);
```

(But I can also live without it. But do please move the color definition to `themes/default-theme.scss` instead of here)
| dark | light
|-------|-----|
|![imagen](https://user-images.githubusercontent.com/5364255/155358301-f87fe53d-f9d6-4189-9516-daf75ab59b2b.png)|![imagen](https://user-images.githubusercontent.com/5364255/155358342-f4dab4fe-f974-43b3-b9a6-b04d097d3320.png)|